packages/trader/src/Assets/Trading/Categories/accumulator-description-video.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { getUrlBase, isMobile } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+
+const AccumulatorDescriptionVideo = ({ selected_contract_type }: { selected_contract_type: string }) => {
```

Comment:
Should this be a generic component, rather than accumulator specific?

Reply:
yeah